src/widgets/scrollbar.rs

```diff
@@ -1128,4 +1131,341 @@ mod tests {
             assert_buffer_eq!(buffer, Buffer::with_lines(expected.clone()));
         }
     }
+
+    #[rstest]
+    #[case("██", 0, 2, "position_0")]
+    #[case("██", 1, 2, "position_1")]
```

Comment:
Instead of having generic cases and pass in an `assertion_message` (which I guess is only to debug which case failed), cases could be [named](https://docs.rs/rstest/latest/rstest/attr.rstest.html#optional-case-description).

EDIT: That's also a better description when the test succeeds.

Reply:
If we use named tests this makes it uneven, e.g.:

<img width="406" alt="image" src="https://github.com/ratatui-org/ratatui/assets/1813121/639eeceb-9015-4492-94ef-618e60a23fbe">

Reply:
yeah, the alignment is good to have for very visual style tests like this.

Reply:
Hm I see ... too bad. Your call what you want to do.